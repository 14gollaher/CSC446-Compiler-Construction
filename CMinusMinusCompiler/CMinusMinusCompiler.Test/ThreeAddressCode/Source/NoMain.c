foo main()
{
	return -1;
}