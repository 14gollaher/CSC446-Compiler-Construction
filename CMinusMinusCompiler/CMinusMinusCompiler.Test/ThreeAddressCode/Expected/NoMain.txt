ERROR: No required function "main" exists
